cmd/server.go

```diff
@@ -12,27 +12,28 @@ import (
 
 	"github.com/ethereum/go-ethereum/crypto"
 
-	"github.com/chainflag/eth-faucet/internal/chain"
-	"github.com/chainflag/eth-faucet/internal/server"
+	"github.com/LiskHQ/lsk-faucet/internal/chain"
+	"github.com/LiskHQ/lsk-faucet/internal/server"
 )
 
 var (
-	appVersion = "v1.1.0"
-	chainIDMap = map[string]int{"goerli": 5, "sepolia": 11155111}
+	appVersion   = "v1.1.0"
+	chainIDMap   = map[string]int{"sepolia": 4202}
+	tokenAddress = flag.String("token-address", os.Getenv("LSK_TOKEN_ADDRESS"), "Contract address of LSK token on the Lisk L2")
 
 	httpPortFlag = flag.Int("httpport", 8080, "Listener port to serve HTTP connection")
 	proxyCntFlag = flag.Int("proxycount", 0, "Count of reverse proxies in front of the server")
 	versionFlag  = flag.Bool("version", false, "Print version number")
 
-	payoutFlag   = flag.Int("faucet.amount", 1, "Number of Ethers to transfer per user request")
+	payoutFlag   = flag.Int("faucet.amount", 1, "Number of Lisk to transfer per user request")
 	intervalFlag = flag.Int("faucet.minutes", 1440, "Number of minutes to wait between funding rounds")
-	netnameFlag  = flag.String("faucet.name", "testnet", "Network name to display on the frontend")
-	symbolFlag   = flag.String("faucet.symbol", "ETH", "Token symbol to display on the frontend")
+	netnameFlag  = flag.String("faucet.name", "sepolia", "Network name to display on the frontend")
+	symbolFlag   = flag.String("faucet.symbol", "LSK", "Token symbol to display on the frontend")
```

Comment:
i think if we want to get the flag as input, then probably it make more sense to make it generic to ERC20?

Reply:
Yes, we could make it generic to ERC20 tokens. I prefer to do those changes in the separate PR, is it fine with you?